src/components/GraphVisualization/SimpleGraph.tsx

```diff
@@ -0,0 +1,87 @@
+import { Node } from "./Node";
+
+function maxValue(arr: number[]) {
+  let max = arr[0];
+
+  arr.forEach((val) => {
+    if (val > max) {
+      max = val;
+    }
+    return null;
+  });
+  return max;
+}
```

Comment:
`Math.max(...numbers)`
![image](https://user-images.githubusercontent.com/21079319/119720814-bfe1f700-be6a-11eb-9e6f-34e01e22377f.png)

Trzy kropki - to operator, robi spread listy. Odpowiednik `*[]` z python'a